generated/nidcpower/README.rst

```diff
@@ -42,25 +42,13 @@ Author       NI
 About
 =====
 
-The **nimi-python** repository generates Python bindings (Application Programming Interface) for interacting with the Modular Instrument drivers. The
-following drivers are supported:
+This package, which is maintained in the `**nimi-python** repository <https://github.com/ni/nimi-python>`_, provides a Python API for the associated driver.
```

Comment:
can we say "for NI-DCPower" instead of "for the associated driver"?
Elsewhere we use " Python module" instead of package. Which is more correct?
The nimi-python blurb is less important. So we should reword:

> The `nidcpower` package provides a Python API for NI-DCPower. The code is maintained in the Open Source repository for `**nimi-python** repository <https://github.com/ni/nimi-python>`.

Reply:
>Elsewhere we use " Python module" instead of package. Which is more correct?

The answer to that may be complicated.

A [module](https://docs.python.org/3/tutorial/modules.html#modules) is basically a .py file, most of the time.

A [package](https://docs.python.org/3/reference/import.html#packages) is *usually a folder with an `__init__.py` file in it. There's another type called a namespace package which can have files all over the place. A package is also considered a module, which makes things confusing. I would say a package is a module when you're importing it.

So the thing you import is a module and the thing you install is a package, at least in my eyes.

Reply:
I'll use "module" for the sake of consistency.